src/rgw/rgw_data_sync.cc

```diff
@@ -1366,32 +1356,17 @@ class RGWDataSyncShardCR : public RGWCoroutine {
         }
 
 
-        yield call(new RGWReadRemoteDataLogShardInfoCR(sync_env, shard_id, &shard_info));
-        if (retcode < 0) {
-          ldout(sync_env->cct, 0) << "ERROR: failed to fetch remote data log info: ret=" << retcode << dendl;
-          stop_spawned_services();
-          drain_all();
-          return set_cr_error(retcode);
-        }
-        datalog_marker = shard_info.marker;
-        remote_trimmed = RemoteNotTrimmed;
 #define INCREMENTAL_MAX_ENTRIES 100
-	ldout(sync_env->cct, 20) << __func__ << ":" << __LINE__ << ": shard_id=" << shard_id << " datalog_marker=" << datalog_marker << " sync_marker.marker=" << sync_marker.marker << dendl;
-	if (datalog_marker > sync_marker.marker) {
+	      ldout(sync_env->cct, 20) << __func__ << ":" << __LINE__ << ": shard_id=" << shard_id << " sync_marker=" << sync_marker.marker << dendl;
+	      {
```

Comment:
it looks like you introduced this extra {} block of nesting that the last commit `rgw: fix up indentation in incremental data sync` 666bfe3a909ec83a3067398e7896527f99283e0a was supposed to remove. while that makes this backport's diff smaller, i worry that it will make future backports in this function more difficult

Reply:
I guess you are right. Diff looks no bigger, but better :)